packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -409,6 +418,11 @@ export const preparePortfolioKit = (
         },
       },
       reader: {
+        getLocalAccount(): LocalAccount {
+          const { accounts } = this.state;
+          const { lca } = accounts.get('agoric') as AccountInfoFor['agoric'];
+          return lca;
```

Comment:
See if this works; I think I was doing this at some point:

```suggestion
          const info = accounts.get('agoric');
          assert.equal(info.type, 'agoric');
          const { lca } = info;
          return lca;
```

Reply:
Thanks for the inspiration.  I ended up using slightly differet code, but as with your suggestion, no Typescript type assertions.